nepalingo-web/src/pages/Login.tsx

```diff
@@ -80,6 +80,10 @@ const Login: React.FC = () => {
     setPassword("");
   };
 
+  const PasswordReset = () => {
```

Comment:
as the function is just navigationg it would be convinient to  just be called in the onClick